src/preparers.py

```diff
@@ -58,6 +58,37 @@ def prepare_commitpackft(row):
     )
 
 
+def prepare_cobra_frames(row):
```

Comment:
Could you leave a few comments to say what this is doing? hard to understand the formatting that is happening here at first glance

Reply:
added!